### PR TITLE
Fix upgrades attributes for traveler-electro

### PR DESCRIPTION
**Pull request by Nielioo**

### CODE REVIEW

File: assets/data/characters/traveler-electro/en.json [hunk 1]
```diff
@@ -12,19 +12,102 @@
...
+        },
+        {
+          "name": "Plunge DMG",
+          "value": "63.9%"
+        },
+        {
+          "name": "Low / High Plunge DMG",
+          "value": "128% / 160%"
+        }
+      ],"type": "NORMAL_ATTACK"
```
Comment (JMPJNS) [inline assets/data/characters/traveler-electro/en.json]: add newline after ],

File: assets/data/characters/traveler-electro/en.json [hunk 2]
```diff
@@ -12,19 +12,102 @@
...
+        },
+        {
+          "name": "Duration",
+          "value": "6 / 15s"
+        },
+        {
+          "name": "CD",
+          "value": "13.5s"
+        }
+      ],"type": "ELEMENTAL_SKILL"
```
Comment (JMPJNS) [inline assets/data/characters/traveler-electro/en.json]: add newline

File: assets/data/characters/traveler-electro/en.json [hunk 3]
```diff
@@ -12,19 +12,102 @@
...
+        },
+        {
+          "name": "CD",
+          "value": "20s"
+        },
+        {
+          "name": "Energy Cost",
+          "value": "80"
+        }
+      ],"type": "ELEMENTAL_BURST"
```
Comment (JMPJNS) [inline assets/data/characters/traveler-electro/en.json]: add newline

**Review by JMPJNS — DISMISSED** [review]

**Comment by man90es** [comment]
Another PR with the same data has been merged, but looks like this one has more correct values for some of the stats

**Review by M4GICB — APPROVED** [review]

**Pull request merged**